v2/tools/generator/internal/codegen/pipeline/add_cross_resource_references.go

```diff
@@ -20,7 +20,7 @@ import (
 // TransformCrossResourceReferencesStageID is the unique identifier for this pipeline stage
 const TransformCrossResourceReferencesStageID = "transformCrossResourceReferences"
 
-var armIDDescriptionRegex = regexp.MustCompile("(?i)(.*/subscriptions/.*?/resourceGroups/.*|ARM ID|Resource ID|resourceId)")
```

Comment:
minor: Consider either colocating this regex in the pipeline stage, or putting the new regex you added in the pipeline stage here?

It feels like the `armIDDescriptionRegex` and the `armIDNameRegex` should be together.

Reply:
Moved.